overlay/files/system/oem/02_agent.yaml

```diff
@@ -1,11 +1,11 @@
 name: "Start agent"
 stages:
   boot:
-    - if: '[ ! -f "/run/cos/recovery_mode" ]'
+    - if: '[ ! -f "/run/cos/recovery_mode" ] && [ -d "/usr/share/systemd" ]'
```

Comment:
avoid running it on openrc systems. We need a better way to check this, probably via the `kairos-agent state get init` or something similar